### PR TITLE
update multibuild and test with xenial images

### PR DESCRIPTION
**Pull request by mattip**
multibuild now supports testing with xenial images. We could update across the board, this is a bit more conservative to see that everything is OK

### CODE REVIEW

**Comment by mattip** [comment]
I am not even sure we use `install_run` on this repo ...

File: .travis.yml [hunk 1]
```diff
@@ -51,35 +51,40 @@ matrix:
...
         - MB_ML_VER=2014
-    # causes a segfault
-    # - os: linux
-    #  arch: arm64
-    #  env:
-    #    - PLAT=aarch64
-    #    - INTERFACE64=1
-    #    - MB_ML_VER=2014
+        - DOCKER_TEST_IMAGE=multibuild/xenial_{PLAT}
+     - os: linux
```
Comment (tylerjereddy) [inline .travis.yml]: There's an extra space before the `- os` that is causing the Travis CI yml parser to fail, so Travis CI never runs for this PR. Checked by pasting into their linter here: https://config.travis-ci.com/explore

Comment (mattip) [reply .travis.yml]: fixing

**Comment by mattip** [comment]
Yes, it should fix the "ContextManager" problem.

**Comment by tylerjereddy** [comment]
I restarted the two failing Travis CI matrix entries. One was a "no output received timeout" & the other was the s390x "no space left" issue we've seen from time to time.

If this becomes all green after the restarts I'll try to merge promptly.

File: .travis.yml [hunk 2]
```diff
@@ -51,35 +51,40 @@ matrix:
...
-    #    - INTERFACE64=1
-    #    - MB_ML_VER=2014
+        - DOCKER_TEST_IMAGE=multibuild/xenial_{PLAT}
+    - os: linux
+      arch: arm64
+      env:
+        - PLAT=aarch64
+        - INTERFACE64=1
+        - MB_ML_VER=2014
+        - DOCKER_TEST_IMAGE=multibuild/xenial_{PLAT}
```
Comment (tylerjereddy) [inline .travis.yml]: This job is still segfaulting--should we disable for now & perhaps try again when we bump up to `0.3.9` or beyond?

Comment (mattip) [reply .travis.yml]: yes. Should we report to upstream?

Comment (tylerjereddy) [reply .travis.yml]: If it keeps happening after I rebase the 0.3.9 bump I'll report it yeah

**Comment by tylerjereddy** [comment]
CI all green now, looks good, thanks Matti.

**Pull request merged**